OSDOCS 10594: Updated the user defined labels and tags for GCP section with a note, and other changes (#73001)
openshift/openshift-docs: modules/installing-gcp-user-defined-labels-and-tags.adoc

User-defined labels and tags are not supported for {product-title} clusters upgraded to {product-title} {product-version}.
====

[NOTE]
====

> Tags once added cannot be updated. Also, a new tag-supported resource creation would fail if configured tags are deleted (key or value).

Author: Ack.

> After adding tags, they cannot be updated. Also, a new tag-supported resource creation would fail if configured tags are deleted (key or value).

Reviewer: ```suggestion
You cannot update the tags that are already added. Also, a new tag-supported resource creation fails if the configured tag keys or tag values are deleted.
```

Author: Thanks, Ack...

.User-defined tags

> User-defined tags are attached to resources created by the {product-title} Installer, Image Registry Operator, Machine API Operator and not on the resources created by any other Operators or the Kubernetes in-tree components.

Author: ACK.

> User-defined tags are attached to resources created by the {product-title} Installer, Image Registry Operator, and Machine API Operator. User-defined tags are not attached to resources created by any other Operators or the Kubernetes in-tree components.

Reviewer: ```suggestion
User-defined tags are attached to resources created by the {product-title} installation program, Image Registry Operator, and Machine API Operator. User-defined tags are not attached to the resources created by any other Operators or the Kubernetes in-tree components.
```

Author: Ack.

User-defined tags are available on the following GCP resources:

* Storage bucket
* Compute Instances

> * Compute Disks

Author: ack
